db/upgrade.php

```diff
@@ -144,5 +133,16 @@ function xmldb_block_configurable_reports_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2019062001, 'block', 'configurable_reports');
     }
 
-    return true;
-}
\ No newline at end of file
+    if ($oldversion < 2022122000) {
+
+        $table = new xmldb_table('block_configurable_reports');
+
+        $field = new xmldb_field('remote', XMLDB_TYPE_INTEGER, '1', XMLDB_UNSIGNED, null, null, '0', null);
+        if (!$dbman->field_exists($table, $field)) {
+            $dbman->add_field($table, $field);
+        }
+        upgrade_plugin_savepoint(true, 2022122000, 'block', 'configurable_reports');
+  }
```

Comment:
There seems to be a difference of indentation here? (closing brace not aligning with the start of the block)